problems/5.loops.js

```diff
@@ -2,17 +2,29 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
+const evenNumbers = (num) => {
+  let newArray = 0
+  for (let i = 0; i <= number; i++)
+  if (num[i] % 2 === 0){
```

Comment:
num is number not an array. So there is no num[i] just like there's no 5[0]. Instead you need to be checking if your i is even. Pro tip increment you i in the for loop i += 2 instead. Then no condition is needed.